middleware/Authorize.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next) => {
+    var token = req.headers['authorization'].split(' ')[1];
```

Comment:
Can you explain this line "req.headers['authorization'].split(' ')[1]"

Reply:
The header is an object that look like this:
{
    "authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
}

Reply:
The second part is the jwt token needed to authorize the user

Reply:
We need the second part (jwt token) to verify the user

Reply:
Excellent, I understand now